app/lib/shared/configuration.dart

```diff
@@ -185,8 +188,14 @@ class EnvConfig {
 /// Configuration from the environment variables.
 final EnvConfig envConfig = EnvConfig._detect();
 
-auth.ServiceAccountCredentials _loadCredentials([String path]) {
-  path ??= envConfig.gcloudKey;
-  final content = File(path).readAsStringSync();
-  return auth.ServiceAccountCredentials.fromJson(content);
+auth.ServiceAccountCredentials _loadCredentials() {
```

Comment:
How does this work in production? there is no `GCLOUD_KEY` variable...

Reply:
It has a different code path:
https://github.com/dart-lang/pub-dartlang-dart/blob/master/app/bin/service/frontend.dart#L166-L175

The issue is that without this change, it will throw an exception regardless of that check.